src/app/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.scss

```diff
@@ -1,22 +1,22 @@
 @import '../../../../assets/stylesheets/base';
-codebases-item-workspaces { display: flex; }
+
+codebases-item-workspaces {
+  display: flex;
+}
+
 .busy-indicator {
-  position: relative;
   display: flex;
   height: auto;
-  width: 25px;
   margin-left: 10px;
+  position: relative;
```

Comment:
@dlabrecq @joshuawilson so the PatternFly code guide is the same code guide that I attributed the changes to, but PatternFly forked it and has not updated to the latest version.